### DISCUSSION TITLE
create subgroup of farm

### DISCUSSION THREAD

**Discussion by mebrahiimi** [OpenFAST/openfast#2476] [General]
Hello,
I am following an approach to subgroup the wind farm. The idea is if we can generate the same result from simulating the whole farm and the subgroup of the farm.
I ran a simulation with 10 inline wind turbine, and I took the .vtk file before the forth WT. next, I generated a .wnd wind file as inflow file for fast.farm trying to simulate the 4th, 5th, 6th wind turbines in one simulation and have similar results. 
As you can see, I could map the 4th wind turbine hub-height wind speed (The black is 10WT simulation, red is 3WT simulation, and we are talking about the second subgroup including (4th,5th,6th) wind turbines), but for the second and third I can see the trend is ok but there is an offset between results.
![wind](https://github.com/user-attachments/assets/2c0f2171-86e2-46ed-b81e-93ddd78918b0)
I would be thankful, if you share some idea with me.

**Comment by jjonkman** [comment]
Dear @majidpoli,

I presume you are using FAST.Farm for these simulations?  If I understand correctly, you are extracting the inflow in front of the 4th turbine from the 10-turbine simulation and feeding that as inflow to a 3-turbine simulation using `Mod_AmbWind` = 2 or 3; is that correct, and if so, is it 2 or 3?

Can you clarify, then why the inflow doesn't match better for the 1st turbine of 3 (4th turbine of 10)?

One issue I foresee is that `Mod_AmbWind` = 2 or 3 makes use of the Taylor's frozen turbulence assumption to propagate the inflow.  However, in FAST.Farm, the wake from a given turbine will recover with increasing downstream distance, which is not consistent with Taylor's frozen turbulence assumption, so, I wouldn't expect that you'd be able to match for the 2nd and 3rd turbines of 3 (5th and 6th turbines of 10).

Best regards,

**Reply by mebrahiimi** [reply]
Dear @jjonkman 
I'm using the Mob_AmbWind = 2. My model has a high resolution grid, so I put a distance more than the half of the length of high-resolution grid in x direction (x = 150 m), so that I can respect the same distance in the second simulation. Therefore, the wake recovery in this 150 m is the reason of the red plot being slightly lower than the black one in the first figure.

Best Regards,

**Reply by jjonkman** [reply]
Thanks for clarifying.
